plugins/reindex/src/main/java/org/elasticsearch/plugin/reindex/AbstractBulkByScrollRequest.java

```diff
@@ -72,6 +74,17 @@
      */
     private WriteConsistencyLevel consistency = WriteConsistencyLevel.DEFAULT;
 
+    /**
+     * Initial delay after a rejection before retrying request. Defaults to 50ms. With the default maxRetries to total time to retry
+     * rejects is about 25 seconds per rejection.
```

Comment:
Right. Per batch. We reset the counter every time a batch makes it through. I can rework the comment.